recoding_tv.R

```diff
@@ -0,0 +1,85 @@
+library(tidyverse)
+library(sjmisc)
+library(here)
+
+anes <- read_delim(here("data","anes_timeseries_2016_rawdata.txt"), delim = "|")
+
+clean <- readRDS("clean.rds")
+
+anes_trimmed <- anes %>% 
+  dplyr::select(V161364:V161427) %>%
+  rename("20/20" = V161364,
```

Comment:
@rheuer83 we should keep the same taxonomy: 

- All lower case
- Start with "tv_"
- remove all symbols, e.g. "/" in the 20/20; just change for another underscore ("_")

Reply:
This is already implemented in the code. (via clean_names from the janitor package)

Reply:
I have kept the original names in the initial recoding section for transparency. The clean_names function converts to snake-case and removes symbols...

Reply:
I had no idea you could do that! Let me go back and check it, then.